sdk/ml/azure-ai-ml/azure/ai/ml/_schema/job/base_job.py

```diff
@@ -59,7 +59,7 @@ class BaseJobSchema(ResourceSchema):
             )
         },
     )
-    compute = ComputeField(required=True)
+    compute = ComputeField(required=False)
```

Comment:
Just curious, is ComputeField optional now?

Reply:
Yes, serverless compute (which is in preview and what we need queue settings for) will be used when compute is not specified

Reply:
Got it. If both compute and queue_settings are none, should we assign defaults for tier & priority or will it be taken care by the backend?

Reply:
It will be taken care of by backend